src/common/certificate-authorities/inject-system-cas.injectable.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { getInjectable } from "@ogre-tools/injectable";
+import { systemCAsInjectorInjectionToken } from "./system-cas-injector-token";
+
+const injectSystemCAsInjectable = getInjectable({
+  id: "inject-system-cas",
+  instantiate: (di) => {
+    const systemCAsInjectors = di.injectMany(systemCAsInjectorInjectionToken);
+
+    return async () => {
+      for (const systemCAsInjector of systemCAsInjectors) {
```

Comment:
Could be `Promise.all()`?

Reply:
I think if we moved to having `.injectable.<platform>.ts` then this would change to be `di.inject(systemCAsInjectorInjectionToken)` and not be "many" at all.